src/Microsoft.Health.SqlServer/Features/Schema/Manager/SchemaClient.cs

```diff
@@ -35,90 +35,121 @@ public void SetUri(Uri uri)
+                .ConfigureAwait(false);
+
             if (response.IsSuccessStatusCode)
             {
 #if NET5_0_OR_GREATER
-                var responseBodyAsString = await response.Content.ReadAsStringAsync(cancellationToken).ConfigureAwait(false);
+                var responseBodyAsString =
+ await response.Content.ReadAsStringAsync(cancellationToken).ConfigureAwait(false);
 #else
-                var responseBodyAsString = await response.Content.ReadAsStringAsync().ConfigureAwait(false);
+                var responseBodyAsString = await response.Content
+                    .ReadAsStringAsync()
+                    .ConfigureAwait(false);
 #endif
                 return JsonConvert.DeserializeObject<List<CurrentVersion>>(responseBodyAsString);
             }
-            else
-            {
-                throw new SchemaManagerException(string.Format(CultureInfo.InvariantCulture, Resources.CurrentDefaultErrorDescription, response.StatusCode));
-            }
+
+            throw new SchemaManagerException(string.Format(
+                CultureInfo.InvariantCulture,
+                Resources.CurrentDefaultErrorDescription,
+                response.StatusCode));
         }
 
         public async Task<string> GetScriptAsync(Uri scriptUri, CancellationToken cancellationToken)
         {
-            HttpResponseMessage response = await _httpClient.GetAsync(scriptUri, cancellationToken).ConfigureAwait(false);
+            HttpResponseMessage response = await _httpClient
+                .GetAsync(scriptUri, cancellationToken)
+                .ConfigureAwait(false);
+
             if (response.IsSuccessStatusCode)
             {
 #if NET5_0_OR_GREATER
                 return await response.Content.ReadAsStringAsync(cancellationToken).ConfigureAwait(false);
 #else
-                return await response.Content.ReadAsStringAsync().ConfigureAwait(false);
+                return await response.Content
+                    .ReadAsStringAsync()
+                    .ConfigureAwait(false);
 #endif
             }
-            else
-            {
-                throw new SchemaManagerException(string.Format(CultureInfo.InvariantCulture, Resources.ScriptNotFound, response.StatusCode));
-            }
+
+            throw new SchemaManagerException(string.Format(
+                CultureInfo.InvariantCulture,
+                Resources.ScriptNotFound,
+                response.StatusCode));
         }
 
         public async Task<CompatibleVersion> GetCompatibilityAsync(CancellationToken cancellationToken)
         {
-            HttpResponseMessage response = await _httpClient.GetAsync(KnownRoutes.RootedCompatibilityUri, cancellationToken).ConfigureAwait(false);
-            if (response.IsSuccessStatusCode)
+            HttpResponseMessage response = await _httpClient
+                .GetAsync(KnownRoutes.RootedCompatibilityUri, cancellationToken)
+                .ConfigureAwait(false);
+
+            if (!response.IsSuccessStatusCode)
```

Comment:
as much as possible, we prioritize handling successful conditionals - can you change this block back to the true case?